src/task6/task6

```diff
@@ -0,0 +1,17 @@
+function reverseArray1(array) {
+    reversedArray = new Array(array.length);
```

Comment:
**var** or **let** or **const** is it ...  ?
And **new Array** is one of the ways to create an array, but **= []** is much less complicated of them